samples/subsys/ipc/openamp/README.md

```diff
@@ -0,0 +1,72 @@
+OpenAMP Example Application
+===========================
+
+Overview
+--------
+
+This application demostrates how to use OpenAMP with Zephyr. It is designed to
+be an OpenAMP counterpart to the RPMsg-Lite example for the LPCXpresso5411x
+chip, to be used for comparison and benchmarking of the two RPC libraries.
+
+Building this application
+-------------------------
+
+Building and running this application requires that libmetal and OpenAMP is
+compiled first. Clone the libmetal and OpenAMP repositories in an appropriate
+location:
+
+    $ git clone https://github.com/OpenAMP/libmetal.git
+    $ git clone https://github.com/OpenAMP/open-amp.git
+
+Compile and build libmetal for both cores:
+
+    $ cd libmetal
+    $ mkdir build-master build-remote
+    $ cd build-master
+    $ cmake -DWITH_ZEPHYR=ON -DBOARD=lpcxpresso54114 ..
+    $ make
+    $ cd ../build-remote
+    $ cmake -DWITH_ZEPHYR=ON -DBOARD=lpcxpresso54114_m0 ..
+    $ make
+
+Compile and build OpenAMP for both cores:
+
+    $ cd open-amp
+    $ mkdir build-master build-remote
+    $ cd build-master
+    $ cmake -DWITH_ZEPHYR=ON -DWITH_PROXY=OFF -DBOARD=lpcxpresso54114 \
+        -DLIBMETAL_INCLUDE_DIR=<path to libmetal>/build-master/lib/include \
+        -DLIBMETAL_LIB=<path to libmetal>/build-master/lib/libmetal.a ..
+    $ make
+    $ cd ../build-remote
+    $ cmake -DWITH_ZEPHYR=ON -DWITH_PROXY=OFF -DBOARD=lpcxpresso54114_m0 \
+        -DLIBMETAL_INCLUDE_DIR=<path to libmetal>/build-remote/lib/include \
+        -DLIBMETAL_LIB=<path to libmetal>/build-remote/lib/libmetal.a ..
+    $ make
+
+Compile the remote application, by going to the toplevel folder of the example
```

Comment:
I think you can delete the ", by going to the..." part of this sentence by the cd change below